platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -59,15 +63,15 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
         this(configurationMigrationService, security, pathBuilder.build());
     }
 
-    ExportCommand(ConfigurationMigrationService configurationMigrationService,
-            Security security, Path path) {
+    ExportCommand(ConfigurationMigrationService configurationMigrationService, Security security,
```

Comment:
For reference, the test: https://sourcegraph.com/github.com/vinamartin/ddf@a67f6a93a16d785608f40f15b17dbd380fdf5870/-/blob/platform/admin/core/admin-core-migration-commands/src/test/java/org/codice/ddf/migration/commands/ExportCommandTest.java#L259